src/core/Flyout.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import {
+  NavigationMenu,
+  NavigationMenuItem,
+  NavigationMenuList,
+  NavigationMenuTrigger,
+  NavigationMenuContent,
+  NavigationMenuViewport,
+  NavigationMenuLink,
+} from "@radix-ui/react-navigation-menu";
+import cn from "./utils/cn";
+
+const NavigationMenuDemo = ({
+  menuItems,
+  navMenuStyling,
+  flyOutStyling,
+}: {
+  menuItems: {
+    label: string;
+    content?: React.ReactNode;
+    link?: string;
+    panelStyling?: string;
+  }[];
+  navMenuStyling?: string;
+  flyOutStyling?: string;
+}) => {
+  const menuLinkStyles =
```

Comment:
It's good thinking having default styling for menu links to suit what you need for the meganav. However since this may have other uses later, I would do what you've done for other aspects of the Flyout and also provide a prop to add custom styles to these as well, i.e. `menuLinkClassName`.

Reply:
Also, I think these default styles need a tweak. They're rounded in the designs, might be other differences too

Reply:
Let's add dark support as well! For every colour, add a corresponding `dark:` class. i.e. `hover:bg-neutral-100` gains a companion of `dark:hover:bg-neutral-1200`. Use the moon/sun icon in the top bar in Storybook to test this easily. As a reminder, all neutral colours should add up to 1300 across variants.

Reply:
We can also move all of these strings outside of the component as constants (i.e. `VIEWPORT_STYLING`), as currently every time the component is rerendered these variables are redefined. This isn't significant as it's just text with no real computation going on (if there was we'd keep it inside and memoise it with useMemo), but every little helps.